### PR TITLE
Fix PlanetShine

### PR DESCRIPTION
**Pull request by Olympic1**
Fixes #2705

### CODE REVIEW

**Comment by valerian** [comment]
Thank you @Olympic1 
This pull request does not have a reason to exist anymore, I'm closing it.

**Comment by valerian** [comment]
Thank you for that, I didn't know it could use the ksp-avc version file automatically, that's great!

**Comment by Olympic1** [comment]
Well it is easier for us to let the author choose what version is compatible with which KSP version.

**Comment by valerian** [comment]
I guess I can also remove the "x_netkan_override" section too?

**Comment by Olympic1** [comment]
No, overrides are usefull if an users wants to play an earlier version of KSP so that CKAN knows which mod version is compatible with that KSP version.

**Comment by valerian** [comment]
this is exactly the same information that I provide in the version file

**Comment by Olympic1** [comment]
But CKAN only knows about that version file when this gets merged, so on earlier versions it technically doesn't exist

**Comment by valerian** [comment]
Ok, so I will do a pull request to remove it later I guess, right?
When do you think this will get merged? Because I might release a new version soon actually.

**Comment by Olympic1** [comment]
Well, the "rule" is that a change needs to be looked over by a human to verify the changes

**Comment by valerian** [comment]
Yes of course.

But my point is that now that the version checking will be entirely automated with data from kerbalstuff + the version file, I should not need to bother you anymore with pull requests for the future versions.

What bothers me is that the x_netkan_override requires to be edited for each new released version of PlanetShine.

Or maybe there is something that I didn't understand.

**Comment by Olympic1** [comment]
Basically what an override stanza does is to override things of a mod's version. You have five variants for this, I'll use `v0.3.7.2` as an example:

* Override everything before `v0.3.7.2`
* Override everything before and equal to `v0.3.7.2`
* Override a specific version, so only change things for `v0.3.7.2`
* Override everything after and equal to `v0.3.7.2`
* Override everything after `v0.3.7.2`

So in this case it will override a specific version (`v0.2.3.1`) and says to CKAN to override the max KSP version of `v0.2.3.1` from 1.0.2 to 1.0.4

**Comment by valerian** [comment]
Ok I see, thank you.

File: NetKAN/PlanetShine.netkan [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
     "license"        : "Apache-2.0",
     "install": [
         {
-            "file": "GameData/PlanetShine/Icons",
-            "install_to": "GameData/PlanetShine"
-        },
-        {
-            "file": "GameData/PlanetShine/Plugins",
+            "find": "GameData/PlanetShine",
             "install_to": "GameData/PlanetShine",
```
Comment (plague006) [inline NetKAN/PlanetShine.netkan]: Just `GameData` or we get `GameData/PlanetShine/PlanetShine`

Comment (Olympic1) [inline NetKAN/PlanetShine.netkan]: Oops, overlooked that

**Pull request merged**